utilities/base_test_case.py

```diff
@@ -1,12 +1,25 @@
+'''
+Contains class for setting up environment to run test and doing clean up after it is finished
+'''
 import unittest
 from selenium import webdriver
 
 
 class BaseTestCase(unittest.TestCase):
+    '''
+    Class for setting up environment to run test and doing clean up after it is finished
+    '''
+
     def setUp(self):
-        self.driver = webdriver.Chrome(executable_path="/Users/atostogan/chromedriver/chromedriver")
+        '''
+        Setup before running test
+        '''
+        self.driver = webdriver.Chrome()
         self.driver.maximize_window()
-        self.driver.get("https://www.google.com")
+        self.driver.get("https://google.com")
 
     def tearDown(self):
+        '''
+        Cleanup after running test
+        '''
         self.driver.close()
```

Comment:
This can fail if setup fails to create a driver. So, you need to:
1. Create a class-level variable `driver = None` outside of setup
2. In tearDown check `if self.driver` and close only if True
3. Also, it is better to use `quite()` to make sure we close ALL windows as close() only closes the active one